app/controllers/users_controller.rb

```diff
@@ -49,7 +49,7 @@ def create
 
   def update
     respond_to do |format|
-      if @user.update_attributes(user_params)
+      if @user.update_attributes(user_params.except(User.non_updateable_params))
```

Comment:
understood :) i've deleted the except method. the github_handle field in a form is disabled, i've added disabled: true in the edit template (commit 78326fc). But when not disabled it displays github_handle can't be changed when trying to change it. shall i leave it disabled?